documentation/automation.md

```diff
@@ -53,11 +55,12 @@ Each directory is used as follows:
 
 ## Utility Tasks and Flows
 
--   **`upgraded_org`** Simulates an org that has been push-upgraded to the latest releases of k12 and underlying dependencies.
+-   **`regression_org`** Simulates an K12 org upgraded from the latest production release to the current beta and its dependencies (EDA), using the unpackaged metadata from the current beta. Use this when you want an upgraded org without needing to make any manual configurations.
+
+-   **`net_new_org`** Simulates the creation of a new K12 org for a new customer, installing the latest beta of K12 and dependencies (EDA). Use this when you want a fully configured regression environment that matches a new installation.
```

Comment:
May want to mention that this flow's output is more like the trial than the installer, but fine as is.